guidelines/releases/release-communication.md

```diff
@@ -0,0 +1,52 @@
+# Release communication
+
+This document describes a release process communication.
+
+## Communication channels
+
+The communication Kyma team should be conducted simultaneously on the two important channels: 
+
+- `c4core-xf-team` - main channel to communicate about the release progress. 
+- `c4core-kyma-scrum-masters` - a channel to escalate actions in case when some team lingers with their job. In that case you can also write directly to the proper Scrum Master.
+
+The Kyma community should be notified about the release on the external channel. Our official channel for external announcements in `#general` on the `kyma-community` slack workspace.
+
+## Communication persons
+
+The Kyma team persons which are responsible for the releasing:
+
+- [Jose Cortina](https://github.com/jose-cortina)
+
+## Release master responsibilities
+
+At the beginning of the release the releasing team should pick a release master which will manage the release process. 
+
+The release master is obliged to do the following things:
+
+- Inform the Kyma team about release deadlines about one week before RC1. The release important dates should be pinned in all channels mentioned above. The Kyma team must know the planned date of the release to don't exceed them. The release master must watch if all Kyma teams respect the deadlines. In case not, he must escalate that to a proper channel or person.
```

Comment:
```suggestion
- Inform Kyma teams about release deadlines on Slack channels at least one week before RC1, and pin the message so that it is easy to find. 
```